data_generators/user_generators.py

```diff
@@ -20,15 +20,17 @@ def __init__(self):
         self.first_name = random_first_name()
         self.last_name = choice(last_names)
         self.email = self.first_name + "." + self.last_name + "@gmail"
+        self.located_in_US = percent_true(100)
```

Comment:
This also needs to be removed since we don't ask this question on the intake forms. Sorry!

Reply:
Removed